dima/react-movies/src/components/searchBar/searchBar.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import SearchFilter from "./searchFilter";
+import "./searchBar.css";
+import ErrorBoundary from "../errorBoundary";
+import * as QueryString from "query-string";
+import FilterProperty from "../../enums/FilterPropery";
+import { RouteComponentProps } from "react-router";
+
+interface SearchBarProps {
+  searchMovies: (searchBarValue: string, searchedBy: string) => void;
+  route: RouteComponentProps;
+  fetchMovies: () => void;
+  currentSortType: string;
+}
+
+interface SearchBarState {
+  searchBarValue: string;
+  searchedBy: string;
+}
+
+export default class SearchBar extends React.Component<
```

Comment:
Pease check the task. When user reloads the page, all filters from URL has to be set. so if searchedBy genre is in url, after page reload it should be on the UI too!